lib/chartjs/chart_helpers.rb

```diff
@@ -49,16 +53,9 @@ def chart(klass, data, options)
 
             #{legend if generate_legend}
           };
-          /* W3C standard */
-          if (window.addEventListener) {
-            window.addEventListener("load", initChart, false);
-            document.addEventListener("page:load", initChart, false);
-          }
-          /* IE */
-          else if (window.attachEvent) {
-            window.attachEvent("onload", initChart);
-            document.attachEvent("page:load", initChart);
-          }
+          
+          initChart();
```

Comment:
i am using the respond_to with javascript for the ajax calls, then i use a example.js.erb with
`$('#partial_area').html("<%= j (render 'partial') %>");`
the gem is writing the javascript to render the chart in this partial, so, the page:load event is not triggered.